package.json

```diff
@@ -130,4 +133,4 @@
     "IE 11"
   ],
   "dependencies": {}
-}
+}
```

Comment:
nit: add end-of-file newline

Reply:
About Active toolbar its not true see attachment
![image](https://user-images.githubusercontent.com/10849982/82788438-c0ffd300-9e68-11ea-8c55-5b7d16dc4d06.png)